scripts/partition_manager/partition_manager.rst

```diff
@@ -328,6 +332,90 @@ It includes :file:`autoconf.h` (which is generated by Kconfig) and uses a Kconfi
 
    #endif /* CONFIG_SOC_NRF9160 */
 
+.. _pm_regions:
+
+Regions
+=======
+The Partition Manager places partitions in different on *regions*.
+Each region is defined by a *start address*, a *size*, a *placement strategy* and conditionally a *device name*.
+Examples of regions are *internal flash* and *external flash*.
+A region only specifies a *device name* if there is a device driver associated with the region, for instance a driver for an external SPI flash.
+A partition sets its region using the ``region`` property in its configuration.
+The internal flash region is used if no region is specified.
+There are three types of *placement strategies*:
+
+#. Start to end
+   Place partitions in a start to end fashion.
+   Partitions stored in a region with this placement strategy can not affect its placement through the ``placement`` keyword.
+   The unused part of the region is exposed through a partition with the same name as the region.
```

Comment:
What does this sentence mean? "exposed" how?

Reply:
exposed through the defines generated by the partition manager script. 

Maybe this would be better:
```
A partition with the same name as the region is created which spans all unused area in the region.
/* or */
The unused part of the region is stored in a partition with the same name as the region.
```